caterer/src/main/java/com/vendor/caterer/interfaces/CatererMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.vendor.caterer.interfaces;
+
+import com.vendor.caterer.dto.CatererCreateRequest;
+import com.vendor.caterer.dto.CatererUpdateRequest;
+import com.vendor.caterer.model.Caterer;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValuePropertyMappingStrategy;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring", nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
```

Comment:
Why you need component model?

Reply:
@Mapper(componentModel = "spring") ensures MapStruct creates a Spring-managed bean for the mapper.